packages/build/bin/compile-package.js

```diff
@@ -185,7 +187,7 @@ function filterArgs(args, filter) {
   const validArgs = [];
   let i = 0;
   while (i < args.length) {
-    const length = filter(args[i]);
+    const length = filter(args[i], args[i + 1]);
```

Comment:
wouldn't `args[i+1]` exceeds the array's index? 
e.g. `args.length = 5`, `i = 4`, `args[5]` doesn't exist

Reply:
you need `args.length -1`

Reply:
> wouldn't `args[i+1]` exceeds the array's index? 
> e.g. `args.length = 5`, `i = 4`, `args[5]` doesn't exist

Yes, `i+1` can exceed the array index. That's fine, `args[i+1]` will be resolved as `undefined` in such case.

```
$ node
> list = [1,2,3]; list[5]
undefined
```